### PR TITLE
DOCSP-35972: Distinct values usage example

### PR DESCRIPTION
**Pull request by norareidy**
Adds a usage example showing how to retrieve distinct field values.
JIRA - https://jira.mongodb.org/browse/DOCSP-35972
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/laravel/DOCSP-35972/usage-examples/distinct/
### Checklist

- [ ] Add tests and ensure they pass
- [ ] Add an entry to the CHANGELOG.md file
- [ ] Update documentation for new features

### CODE REVIEW

File: docs/usage-examples/distinct.txt [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+- Uses the ``Movie`` Eloquent model to represent the ``movies`` collection in the
+  ``sample_mflix`` database
+- Retrieves distinct field values of documents from the ``movies`` collection that match a query filter
+- Prints the distinct values
+
+The example calls the following methods on the ``Movie`` model:
+
+- ``where()``: matches documents in which the value of the ``year`` field is ``1930``.
+- ``select()``: retrieves the matching documents' ``imdb.rating`` field values.
```
Comment (rustagir) [inline docs/usage-examples/distinct.txt:40]: S: is there a better field to select for than a numerical field? Like genre or director maybe?

File: docs/usage-examples/distinct.txt [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can retrieve distinct field values of documents in a collection by calling the ``distinct()``
+method on an object collection or a query builder.
+
+To retrieve distinct field values, pass a query filter to the ``where()`` method and a field name
+to the ``select()`` method. Then, call ``distinct()`` to return the unique field values of documents
+that match the query filter.
```
Comment (rustagir) [inline docs/usage-examples/distinct.txt]:
```suggestion
to the ``select()`` method. Then, call ``distinct()`` to return the unique values of the selected field in documents
that match the query filter.
```

**Review by rustagir — APPROVED** [review]
approving with some non blocking comments, happy to take another look if need be!

File: docs/usage-examples/distinct.txt (hunk 1 above)
Comment (norareidy) [reply docs/usage-examples/distinct.txt:40]: Sure, changed to the directors field

**Review by GromNaN — APPROVED** [review]
LGTM.

Could you make a note to revisit this page when working on the aggregation builder docs?

**Comment by ccho-mongodb** [comment]
> Could you make a note to revisit this page when working on the aggregation builder docs?

@GromNaN 
Sure! What would you like to see on this page once the Aggregation Builder docs are completed?

**Comment by GromNaN** [comment]
> Sure! What would you like to see on this page once the Aggregation Builder docs are completed?

It should demonstrate how to do almost the same result using an aggregation with a `$group` stage.

**Pull request merged**